src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -91,6 +91,19 @@ public void Contains(int size)
         }
 
         [Benchmark]
+        [MemberData(nameof(ContainsStringComparisonArgs))]
+        public void Contains(StringComparison comparisonType, int size)
+        {
+            PerfUtils utils = new PerfUtils();
+            string testString = utils.CreateString(size);
+            string subString = testString.Substring(testString.Length / 2, testString.Length / 4);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < 10000; i++)
+                        testString.Contains(subString, comparisonType);
```

Comment:
Perf tests run against the .NET Framework as well, so this will have to be moved to `Per.String.netcoreapp.cs` 

You can run against the .NET Framework using `msbuild /T:BuildAndTest /P:TargetGroup=netfx`

As per the UWP test failures, you'll need to port the API to corert as @jkotas mentioned